lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -1120,7 +1097,56 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
             let epochs = poolGarbageCollectionEpochNo <$> events
             (reverse epochs `zip` [1 ..]) `shouldSatisfy` all (uncurry (==))
 
+    it "STAKE_POOLS_SMASH_01 - fetching metadata from SMASH works with delisted pools" $
+        \ctx -> runResourceT $ do
+            smashUrl <- liftIO $ getEnv "CARDANO_WALLET_SMASH_URL"
```

Comment:
Running this one and STAKE_POOLS_LIST_01 in parallel will likely fail since they'll be both changing smash settings. Let's hope it doesn't happen _too often_.

Reply:
Should be fixed